cap-2.x/gke/providers.tf

```diff
@@ -5,19 +5,19 @@ terraform {
 provider "google" {
   # You can also set GOOGLE_CREDENTIALS to point to the service account key file to pick up the credentials
   #   See https://www.terraform.io/docs/providers/google/provider_reference.html
-  version     = "~> 2.3"
+  version     = "3.48.0"
```

Comment:
Is v3.48 a new requirement? Packaged version is v3.43:

https://build.opensuse.org/package/show/systemsmanagement:terraform/terraform-provider-google

Reply:
Changed back to 3.43.0